README.md

```diff
@@ -20,13 +20,94 @@ Note that the while this repo is named `submission-schema`, the generated artifa
 
 ## Developer Documentation
 
-<details>
-Use the `make` command to generate project artefacts:
+### Updating the submission schema
 
-- `make all`: make everything
-- ~~`make deploy`: deploys site~~
+Here's how you can update the submission schema to use a new version of `nmdc-schema`:
 
-</details>
+1. Update the one occurrence of the `nmdc-schema` version number in `project.Makefile`:
+
+    ```diff
+    local/nmdc.yaml:
+    -        wget -O $@ https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.0.1/nmdc_schema/nmdc_materialized_patterns.yaml
+    +        wget -O $@ https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.1.0/nmdc_schema/nmdc_materialized_patterns.yaml
+    ```
+
+2. Update all the `nmdc-schema` version numbers in `sheets_and_friends/tsv_in/import_slots_regardless.tsv`
+
+    ```diff
+    - schema_fp: https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.0.1/nmdc_schema/nmdc_materialized_patterns.yaml; source_class: Biosample; destination_class: BuiltEnvInterface
+    + schema_fp: https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.1.0/nmdc_schema/nmdc_materialized_patterns.yaml; source_class: Biosample; destination_class: BuiltEnvInterface
```

Comment:
I think the `schema_fp` stuff actually is from some log output. I'd recommend updating this to match what's in the TSV file (although showing an entire line probably isn't too critical)

```suggestion
    - Biosample	https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.0.1/nmdc_schema/nmdc_materialized_patterns.yaml	abs_air_humidity ...
    + Biosample	https://raw.githubusercontent.com/microbiomedata/nmdc-schema/v11.1.0/nmdc_schema/nmdc_materialized_patterns.yaml	abs_air_humidity ...
```

Reply:
Oops! Yes, you are right. I don't know where I got those `schema_fp` lines (it could have been from some log output, like you suggested). I did mean for this diff snippet to contain a before and after of a TSV line. I will update it accordingly (by accepting/committing the change you suggested here). Thank you!